lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -656,7 +656,7 @@ public function get_item_schema() {
 				'slug'        => array(
 					'description' => 'An alphanumeric identifier for the object unique to its type.',
 					'type'        => 'string',
-					'context'     => array( 'embed', 'view', 'edit' ),
+					'context'     => array( 'view', 'edit' ),
```

Comment:
I feel very strongly that `slug` is a User field I would not expect to be available in the embed context.  The field is available in the `view` and `edit` context.  

Is there a reason besides consistency?